aine/index.php

```diff
@@ -0,0 +1,2 @@
+<?php
```

Comment:
Do we need this anymore?

Reply:
No, we don't need to include this file anymore.